examples/shaders/sky-shader/src/lib.rs

```diff
@@ -175,3 +175,14 @@ pub fn main_vs(#[spirv(vertex_index)] vert_idx: i32, #[spirv(position)] builtin_
 
     *builtin_pos = pos.extend(0.0).extend(1.0);
 }
+
+#[cfg(test)]
+mod tests {
+    // Note this useful idiom: importing names from outer (for mod tests) scope.
```

Comment:
nit: probably don't need this comment, as it's an extremely common pattern known throughout the ecosystem!